css/shop.css

```diff
@@ -0,0 +1,47 @@
+/* School CSS */
```

Comment:
Nope, but I didn't want to mess with this without a reason...it's basically just copied over from icommons_tools.